src/test/ui/feature-gates/feature-gate-const_in_array_repeat_expressions.rs

```diff
@@ -0,0 +1,11 @@
+// ignore-tidy-linelength
+#![allow(warnings)]
+
+struct Bar;
+
+fn foo() {
+    let arr: [Option<String>; 2] = [None::<String>; 2];
+    //~^ ERROR the trait bound `std::option::Option<std::string::String>: std::marker::Copy` is not satisfied [E0277]
+}
+
```

Comment:
Turns out this test was not thorough enough, and [some other cases sneaked past the feature gate](https://github.com/rust-lang/rust/issues/49147#issuecomment-726796665).